executor/union_scan_test.go

```diff
@@ -416,6 +417,14 @@ func TestForApplyAndUnionScan(t *testing.T) {
 }
 
 func TestIssue28073(t *testing.T) {
+	var wg sync.WaitGroup
+	wg.Add(1)
+	dumpChan := make(chan struct{})
+	defer func() {
+		close(dumpChan)
+		wg.Wait()
+	}()
+	go testkit.DebugDumpOnTimeout(&wg, dumpChan, 20*time.Second)
```

Comment:
The timeout issue is fixed. Is it necessary to keep this?

Reply:
Not really, I just removed it from the PR. I may create a separate PR for the 'DebugDumpOnTimeout' function later on.